src/services/oidc-data.service.ts

```diff
@@ -11,7 +11,7 @@ export class OidcDataService {
         headers = headers.set('Accept', 'application/json');
 
         return this.httpClient.get<T>(url, {
```

Comment:
comma too many?

Reply:
no because of git code styling --> use trailing commas